Pipfile

```diff
@@ -7,10 +7,10 @@ name = "pypi"
 arrow = "*"
 celery = "*"
 dj-database-url = "*"
-django = "*"
+django = ">=2.0.8"
```

Comment:
You don't actually have to pin these to a `>=` value btw. If you just run `pipenv update` it would update the `Pipfile.lock` to the latest versions without making any edits to the `Pipfile`. 

(No need to change this, just in case you weren't aware of that feature!)